src/styles/catalog.scss

```
@@ -26,10 +26,18 @@ $productGap: 48px;
   border-radius: 5px;
 
   &__photo {
-    padding-top: 16px;
+    padding: 13px 0;
     display: flex;
     justify-content: center;
-    margin-bottom: 45px;
+    margin-bottom: 29px;
+
+    border: 3px solid transparent;
+    transition: transform 1s, border 0s;
+  }
+
+  &__photo:hover {
+    transform: scale(1.2);
+    border: 3px solid #f3f3f3;
```

Ти вже задав вище border: 3px solid transparent; тому тобі при hover треба лише змінити колір границі з прозорого на необхідний. 
```suggestion
    border-color: #f3f3f3;
```